### DISCUSSION TITLE
console commands not working?

### DISCUSSION THREAD

**Discussion by drfleetles** [GriefPrevention/GriefPrevention#1437] [Q&A]
I am using a spigot 1.17 server and im just testing out the plugin.
 all the commands that need to be issued in the console cannot recognise the command.
 
 my guess is that when i issue in the console. it issues it as a player as such
 `01.07 09:37:04 [Server] Server thread/INFO Unknown command. Type "/help" for help.
01.07 09:37:05 [Multicraft] Dr_Fleetles ran command: /deleteallclaimsinworld `

BTW the return of the command comes first

**Comment by Jikoo** [comment]
GP does not issue console commands as a player. Probably an issue with your wrapper if you see players being loaded to run console commands.
